sys-devel/clang-common/clang-common-18.0.0_pre20230803-r2.ebuild

```
@@ -178,7 +178,9 @@ src_install() {
 		local abi_chost=$(get_abi_CHOST "${abi}")
 
 		local tool
-		for tool in ${abi_chost}-clang{,++,-cpp}; do
+		# Additionally install configuration files for the host tuple with gentoo replaced by unknown, since some
+		# packages force compiling with *-unknown-* tuple.
+		for tool in ${abi_chost}-clang{,++,-cpp} ${abi_chost/gentoo/unknown}-clang{,++,-cpp}; do
```

let's do it only for musl per IRC discussion I suppose:
```suggestion
		for tool in ${abi_chost}-clang{,++,-cpp} $(usev elibc_musl "${abi_chost/gentoo/unknown}-clang{,++,-cpp}"); do
```

does that work?

---

also, if CHOST has unknown already, will this fail in the newinst call because we're creating duplicate files?

---

alright, i'll fix this tomorrow morning.